### PR TITLE
Upgrade kernel from 3.16.59-1 to 3.16.68-2

### PR DESCRIPTION
**Pull request by qiluo-msft**

### CODE REVIEW

**Comment by lguohan** [comment]
retest this please

**Comment by qiluo-msft** [comment]
Retest this please

**Comment by qiluo-msft** [comment]
Built in dev local. PR checker doesn't work right now.

File: Makefile [hunk 1]
```diff
@@ -45,7 +45,7 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	fakeroot make -f debian/rules.gen setup_amd64_none_amd64
 
 	# Applying patches and configuration changes
-	git diff
+	git --no-pager diff
 	git add debian/build/build_amd64_none_amd64/.config -f
```
Comment (yxieca) [inline Makefile]: Good catch!

**Pull request merged**